rllib/agents/a3c/a3c_tf_policy.py

```diff
@@ -119,6 +122,42 @@ def setup_mixins(policy, obs_space, action_space, config):
     LearningRateSchedule.__init__(policy, config["lr"], config["lr_schedule"])
 
 
+def view_requirements_fn_pg(policy: Policy) -> Dict[str, ViewRequirement]:
```

Comment:
Hmm can we also add some documentation for this in the .rst files? (rllib-concepts)?

Reply:
+1